tools/entity_positions/epos.c

```diff
@@ -6,9 +6,8 @@
 #include "ext/c_header_parser/parser.h"
 
 /* CSV Format Example:
-Adv2,ENEMIES,,,,,
-12345,67890,KIKI,0,0,0,0
-45678,87445,MON,-5,-8,5,10
+Adv2,ENEMIES,25,10,,,,,
+20,0,10,5,KIKI,0,0,0,0
```

Comment:
I think I'm ok with the idea that the CSV contains Enums which refer to the enemy, however can it accept both?

Reply:
What do you mean?
The Entity Type is there so the program knows how the `#define` names need to be parsed.
(There's only ENEMY__KIKI, not ITEM__KIKI, for example)

Reply:
Can it accept both a name reference (`KIKI`) but also an id (`1`)?

My opinion is that this tool should not care if an enemy is valid. We want this interchange format to be used by people who may be adding enemies.